plugins/source/gitlab/resources/services/users/users_fetch.go

```diff
@@ -0,0 +1,40 @@
+package users
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/plugins/source/gitlab/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/xanzy/go-gitlab"
+)
+
+func fetchUsers(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	c := meta.(*client.Client)
+
+	opt := &gitlab.ListUsersOptions{
+		ListOptions: gitlab.ListOptions{
+			PerPage: 1000,
+			Page:    0,
+		},
+	}
+	for {
+		// Get the first page with projects.
+		users, resp, err := c.Gitlab.Users.ListUsers(opt)
+		if err != nil {
+			return err
+		}
+		if len(users) == 0 {
+			return nil
```

Comment:
can the next page be still present?
maybe leave the login for loop end only for the next page check?